ONSdigital/census-rh-ui
# CR-1791 BFPO Regex Change

Removed BFPO part of regex, and added 'BF' as valid start letters

Signed-off-by: Paul-Joel <paul.joel@ons.gov.uk>

# Motivation and Context
Update to remove BFPO for postcode regex in prevent AIMS error

# What has changed
Changes to existing regex

No Cucumber updates required

# How to test?
Check that a BFPO postcode is rejected in RH as invalid
Check that a BF postcode (BF1 3AG) is still rejected

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1791
